catalog-info.yaml

```diff
@@ -1115,6 +1115,41 @@ spec:
         everyone:
           access_level: BUILD_AND_READ
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: beats-aws-tests-daily
```

Comment:
I recommend we don't add the schedule (`-daily`) in the pipeline **name**: for now, indeed, it is designed to be triggered via a schedule from the corresponding scheduler pipeline, but, who knows, in the future maybe we also schedule it to run, say, upon merge commit or even PR.

So I recommend we keep the name agnostic of the schedule.